as/plugins/org.jboss.tools.as.rsp.ui/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion> 
+	<parent>
+		<groupId>org.jboss.tools.as</groupId>
+		<artifactId>plugins</artifactId>
+		<version>3.10.3-SNAPSHOT</version>
+	</parent>
+	<groupId>org.jboss.tools.as.plugins</groupId>
+	<artifactId>org.jboss.tools.as.rsp.ui</artifactId> 
+	<packaging>eclipse-plugin</packaging>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>get-libs</id>
+						<goals>
+							<goal>copy</goal>
+						</goals>
+						<phase>generate-resources</phase>
+					</execution>
+				</executions>
+				<configuration>
+					<skip>false</skip>
+					<outputDirectory>${basedir}/lib/</outputDirectory>
+					<!-- baseVersion is to avoid SNAPSHOT dependencies being copied with 
+						ever daily changing timestamp -->
+					<useBaseVersion>true</useBaseVersion>
+					<artifactItems>
+						<artifactItem>
+							<groupId>org.eclipse.lsp4j</groupId>
+							<artifactId>org.eclipse.lsp4j.jsonrpc</artifactId>
+							<version>0.9.0</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jboss.tools.rsp.api.bundles</groupId>
+							<artifactId>org.jboss.tools.rsp.api</artifactId>
+							<version>0.25.0.Final</version>
+						</artifactItem>
+						<artifactItem>
+						    <groupId>com.google.code.gson</groupId>
+						    <artifactId>gson</artifactId>
+						    <version>2.10.1</version>
```

Comment:
there is already a gson bundle in the tp : https://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.28.0.Final-SNAPSHOT/REPO/plugins/com.google.gson_2.10.1.v20230109-0753.jar